components/kitchen-sink.tsx

```diff
@@ -115,20 +118,21 @@ export const KitchenSink: React.FC = (): JSX.Element => {
                 <View style={{ justifyContent: 'center', marginHorizontal: 24, marginVertical: 24 }}>
                     <H5>Appbar</H5>
                     <View style={{ marginTop: 24 }}>
-                        <Appbar.Header style={{ width: 350 }}>
-                            <Appbar.Content title="Title" subtitle="Subtitle" />
-                        </Appbar.Header>
-                        <Appbar.Header style={{ width: 350 }}>
-                            <Appbar.Content title="Title" subtitle="Subtitle" />
-                            <Appbar.Action icon="magnify" />
-                            <Appbar.Action icon="dots-vertical" />
-                        </Appbar.Header>
-                        <Appbar.Header style={{ width: 350 }}>
-                            <Appbar.BackAction />
-                            <Appbar.Content title="Title" subtitle="Subtitle" />
-                            <Appbar.Action icon="magnify" />
-                            <Appbar.Action icon="dots-vertical" />
-                        </Appbar.Header>
+                        <Appbar theme={theme.dark ? blueDarkAlt : {}} style={{ marginBottom: 16 }}>
```

Comment:
Should we be making a wrapper for any component that use this `theme.dark ? blueDarkAlt : {}` logic?

Reply:
Using wrappers will make the main application code much more readable and less error prone (e.g., by forgetting to put the theme logic on every instance of a particular component).

Reply:
I was having a hard time using wrapper components for certain RNP components, but that may have been because of the deep clone issue. I'll take a second look at adding these.